example/spec/parallel/createReconciliationReport/CreateReconciliationReportSpec.js

```diff
@@ -554,6 +560,12 @@ describe('When there are granule differences and granule reconciliation is run',
     // report record in db and report in s3
     let reportRecord;
     let report;
+    let internalReportAsyncOperationId;
+
+    afterAll(async () => {
+      await deleteAsyncOperation({ prefix: config.stackName, asyncOperationId: internalReportAsyncOperationId });
```

Comment:
I'm wondering if we want to check for `internalReportAsyncOperationId` before calling `deleteAsyncOperation`.

```
if (internalReportAsyncOperationId) {
  await deleteAsyncOperation({ prefix: config.stackName, asyncOperationId: internalReportAsyncOperationId });
}
```
It doesn't matter functionally as, if we don't have `internalReportAsyncOperationId` it means that an async operation was not created and therefor won't need be deleted. It might simplify the error logging though. I think right now we'd see an error when we try to `createReconciliationReport` if it fails and then another when we get to the `afterAll`.

Reply:
This would apply to all other instances of the pattern in other tests here.